conf/application.conf

```diff
@@ -1,15 +1,6 @@
 # This is the main configuration file for the application.
 # ~~~~~
 
-# Secret key
-# ~~~~~
-# The secret key is used to secure cryptographics functions.
-#
-# This must be changed for production, but we recommend not changing it in this file.
-#
-# See http://www.playframework.com/documentation/latest/ApplicationSecret for more details.
-play.crypto.secret="Is]F;gOuRS]X[eTv4j9k;_jC3Tc>2Xv<D_On/CRr/MBoJAM5yBB_w2Fp]1c7xDpX"
```

Comment:
there was no need to have this default value, in fact it makes it more likely to accidentally use this (public) value in PROD. 
So i have removed and added the correctly named one for play 2.8 in the relevant secret files.